bestiary/core/schema.py

```diff
@@ -120,8 +122,15 @@ class Meta:
         model = Ecosystem
 
 
+class ProjectType(DjangoObjectType):
+    """Project objects are meant to gather a set of end-point views"""
+
+    class Meta:
+        model = Project
```

Comment:
Checking the type with the graphql interface, it shows a `projectSet` attribute. I guess it will show the list of child projects. Do you think we can rename the field to something more clear? projectSet makes reference to Django.

Reply:
Oops! I wasn't aware of that. Let me check this with more detail.

Reply:
I solved this by adding a related name in the model's field: `related_name='child_projects'`.

This way, the interface is displaying it as expected. However, the help text does not change (it also says `Parent project` in this related field) and I couldn't solve this yet.

Reply:
Shouldn't be better to call it subprojects?

Reply:
I agree, that's better.